rasa/server.py

```diff
@@ -564,7 +542,9 @@ async def append_events(request: Request, conversation_id: Text):
         try:
             async with app.agent.lock_store.lock(conversation_id):
                 processor = app.agent.create_processor()
-                tracker = processor.get_tracker(conversation_id)
+                tracker = await processor.fetch_tracker_with_initial_session(
```

Comment:
I think [this piece of code](https://github.com/RasaHQ/rasa-x/blob/4e2e5a4613eef15816968389be4eb9d616f47fca/src/community/rasax/community/api/blueprints/stack.py#L219) in the Rasa X code base depends on the fact that the tracker is actually empty.

Reply:
I think we should either check if the new events already a starting sequence or not adding the starting sequence. What do you think?